On musl, FILE is not a complete type for C standards >=C11. See:

	https://git.musl-libc.org/cgit/musl/commit/?id=f368d9fd26ae002fe2fce20add4cb2b806f48972

For this reason, the fpurge test fails to compile with:

	fpurge.c: In function 'main':
	fpurge.c:32:21: error: storage size of 'fp_bad' isn't known
	   32 |         static FILE fp_bad;
	      |                     ^~~~~~

diff -upr libbsd-0.11.5.orig/test/Makefile.am libbsd-0.11.5/test/Makefile.am
--- libbsd-0.11.5.orig/test/Makefile.am	2022-02-04 23:14:39.252701411 +0100
+++ libbsd-0.11.5/test/Makefile.am	2022-02-04 23:14:45.209390741 +0100
@@ -38,7 +38,6 @@ check_PROGRAMS = \
 	fgetln \
 	funopen \
 	fparseln \
-	fpurge \
 	md5 \
 	nlist \
 	proctitle-init \
